app/src/androidTest/java/net/skyscanner/backpack/docs/DocsRegistry.kt

```diff
@@ -34,86 +34,92 @@ import org.threeten.bp.LocalDate
+    ViewScreenshot("Card - Corner style large", "corner-style-large"),
+    ViewScreenshot("Card - With divider", "with-divider"),
+    ViewScreenshot("Card - With divider arranged vertically", "with-divider-arranged-vertically"),
+    ViewScreenshot("Card - With divider without padding", "with-divider-without-padding"),
+    ViewScreenshot("Card - With divider and corner style large", "with-divider-and-corner-style-large"),
+    ViewScreenshot("Chip - Default", "all"),
+    ViewScreenshot("Chip - Outline", "outline"),
+    ViewScreenshot("Chip - Custom", "custom"),
+    ViewScreenshot("Chip - With icon", "with-icon"),
+    ViewScreenshot("Checkbox", "default"),
+    ViewScreenshot("Dialog - With call to action", "with-cta", ::setupDialog),
+    ViewScreenshot("Dialog - Delete confirmation", "delete-confirmation", ::setupDialog),
+    ViewScreenshot("Dialog - Flare", "with-flare", ::setupDialog),
+    ViewScreenshot("Flare - Default", "default"),
+    ViewScreenshot("Flare - Pointing up", "pointing-up"),
+    ViewScreenshot("Flare - Pointer offset", "pointer-offset"),
+    ViewScreenshot("Flare - Rounded", "rounded"),
+    ViewScreenshot("Flare - Inset padding mode", "inset-padding"),
+    ViewScreenshot("Horizontal Nav", "default"),
+    ViewScreenshot("Floating Action Button", "default"),
+    ViewScreenshot("Nav Bar - Default", "expanded"),
+    ViewScreenshot("Nav Bar - Default", "collapsed", ::setupNavBarCollapsed),
+    ViewScreenshot("Nav Bar - With Menu", "navigation", ::setupNavBarCollapsed),
+    ViewScreenshot("Nudger", "all"),
+    ViewScreenshot("Overlay", "all"),
+    ViewScreenshot("Panel", "all"),
+    ViewScreenshot("RadioButton", "default"),
+    ViewScreenshot("Rating - Default", "default"),
+    ViewScreenshot("Rating - Horizontal", "sizes"),
+    ViewScreenshot("Rating - Vertical", "vertical"),
+    ViewScreenshot("Rating - Pill", "pill"),
+    ViewScreenshot("Slider", "all"),
+    ViewScreenshot("Snackbar", "default", ::setupSnackbar),
+    ViewScreenshot("Snackbar", "icon", ::setupSnackbarIconAction),
+    ViewScreenshot("Star Rating - Default", "default"),
+    ViewScreenshot("Star Rating Interactive", "default"),
+    ViewScreenshot("Switch", "default"),
+    ViewScreenshot("Text - View - Default", "default"),
+    ViewScreenshot("Text - View - Emphasized", "emphasized"),
+    ViewScreenshot("Text - View - Heavy", "heavy"),
+    ViewScreenshot("Text Field - Default", "default"),
+    ViewScreenshot("Text Field - With labels", "labels"),
+    ViewScreenshot("Text Spans", "default"),
+    ViewScreenshot("Spinner - Default", "default"),
+    ViewScreenshot("Spinner - Small", "small"),
     // Leave toast last as it stays visible in the screen for a while
-    Screenshot("Toast", "default", ::setupToast)
-  ).map { it.toArgs() }
+    ViewScreenshot("Toast", "default", ::setupToast)
+  )
 
   init {
     CalendarDispatchers.setBackground(TestCoroutineDispatcher())
   }
 }
 
-data class Screenshot(
```

Comment:
We only used this class to map it to array of arguments for GenerateScreenshots. It's been replaced with 2 corresponding functions